cookbooks/aws-parallelcluster-install/recipes/ephemeral_drives.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster
+# Recipe:: ephemeral_drives
+#
+# Copyright:: 2013-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+cookbook_file 'setup-ephemeral-drives.sh' do
+  source 'base/setup-ephemeral-drives.sh'
+  path '/usr/local/sbin/setup-ephemeral-drives.sh'
+  user 'root'
```

Comment:
nitpick, seems to not be clear from the [documentation](https://docs.chef.io/resources/cookbook_file/) but this `user` seems to be equal to `owner`. Should we align the fields?